_posts/2017/08/2017-08-11-self-balancing-bot-using-96boards-part1.md

```diff
@@ -122,7 +122,9 @@ run the C program (server) in background and Python program (client) in foregrou
 
 Source code and detailed instructions can be found in 96Boards [projects repository](https://github.com/96boards/projects/blob/master/imu)
 
-Demonstration video can be found in [96Boards Youtube Channel](https://youtu.be/mswTk_Sliiw)
+Demonstration video:
+
+{% include media.html media_url="https://youtu.be/mswTk_Sliiw" %}
```

Comment:
As per the instruction here 
https://github.com/96boards/website/blob/master/help/blog.md#media-slideshare-includes-etc

Youtube need to be in the following format to be embedded properly
https://www.youtube.com/embed/mswTk_Sliiw